patches/0013-start-arm-optimized-nistz256

```diff
@@ -0,0 +1,2577 @@
++                                            ////    movdqa  16*1($in_t), $T0b
++                                            ////    movdqa  16*2($in_t), $T0c
++                                            ////    movdqa  16*3($in_t), $T0d
++                                            ////    movdqa  16*4($in_t), $T0e
++                                            ////    movdqa  16*5($in_t), $T0f
++                                            ////    lea 16*6($in_t), $in_t
++
++    // $TMP0 = ($M0 == $INDEX)? All 1s : All 0s
++    cmeq    $TMP0.4s, $M0.4s, $INDEX.4s     ////    movdqa  $M0, $TMP0
++                                            ////    pcmpeqd $INDEX, $TMP0
++    // Increment $M0 lanes
++    add     $M0.4s, $M0.4s, $ONE.4s         ////    paddd   $ONE, $M0
++
++    // [$T0a-$T0f] := [$T0a-$T0f] AND $TMP0
++    // values read from the table will be 0'd if $M0 != $INDEX
++    // [$Ra-$Rf] := [$Ra-$Rf] OR [$T0a-$T0b]
++    // values in output registers will remain the same if $M0 != $INDEX
++    and     $T0a.16b, $T0a.16b, $TMP0.16b    ////    pand   $TMP0, $T0a
++    and     $T0b.16b, $T0b.16b, $TMP0.16b    ////    pand   $TMP0, $T0b
++    orr     $Ra.16b, $Ra.16b, $T0a.16b       ////    por    $T0a, $Ra
++    orr     $Rb.16b, $Rb.16b, $T0b.16b       ////    pand   $TMP0, $T0c
++                                             ////    por    $T0b, $Rb
++    and     $T0c.16b, $T0c.16b, $TMP0.16b    ////    pand   $TMP0, $T0d
++    and     $T0d.16b, $T0d.16b, $TMP0.16b    ////    por    $T0c, $Rc
++    orr     $Rc.16b, $Rc.16b, $T0c.16b       ////    pand   $TMP0, $T0e
++    orr     $Rd.16b, $Rd.16b, $T0d.16b       ////    por    $T0d, $Rd
++                                             ////    pand   $TMP0, $T0f
++    and     $T0e.16b, $T0e.16b, $TMP0.16b    ////    por    $T0e, $Re
++    and     $T0f.16b, $T0f.16b, $TMP0.16b    ////    por    $T0f, $Rf
++    orr     $Re.16b, $Re.16b, $T0e.16b
++    orr     $Rf.16b, $Rf.16b, $T0f.16b
++
++    // Decrement loop counter; loop back if not 0
++    subs    $Ctr, $Ctr, #1                  ////    dec %rax
++    bne     .Lselect_w5_loop                ////    jnz .Lselect_loop_sse_w5
++
++    // Write [$Ra-$Rf] to memory at the output pointer
++    st1     {$Ra.2d, $Rb.2d, $Rc.2d, $Rd.2d}, [$Val_in],#64
++    st1     {$Re.2d, $Rf.2d}, [$Val_in]     ////    movdqu	$Ra, 16*0($val)
++                                            ////    movdqu	$Rb, 16*1($val)
++                                            ////    movdqu	$Rc, 16*2($val)
++                                            ////    movdqu	$Rd, 16*3($val)
++                                            ////    movdqu	$Re, 16*4($val)
++                                            ////    movdqu	$Rf, 16*5($val)
++	ret
++.size	ecp_nistz256_select_w5,.-ecp_nistz256_select_w5
++___
++}
++########################################################################
++# scatter-gather subroutines
```

Comment:
Can't we remove these functions? They are being replaced by the "select" constant-time functions instead and nothin in AWS-LC should be calling these functions.

Reply:
They are removed. Thanks.